include/crm/msg_xml.h

```diff
@@ -177,7 +177,6 @@ extern "C" {
 #define PCMK_XE_RESOURCES_CONFIGURED        "resources_configured"
 #define PCMK_XE_RESULT_CODE                 "result-code"
 #define PCMK_XE_REVISED_CLUSTER_STATUS      "revised_cluster_status"
-#define PCMK_XE_RKT                         "rkt"
```

Comment:
Technically should be an "API: " commit since the "New PCMK_XE_RKT string constant" one was, but it doesn't really matter

Reply:
I'll make a note to drop the original message from the changelog for the next release